Total Instances: 82

Total Correct: 61
Total Runtime Failed: 8
Total Compilation Failed: 5
Total Test Failed: 7
Total Infinite Loop: 1

Accuracy: 74.39024390243902
Runtime Rate: 9.75609756097561
Compilation Rate: 6.097560975609756
Test Failed Rate: 8.536585365853659
Infinite Loop Rate: 1.2195121951219512

=================================================================================================
Failed Test Files: ['codeforces_96_B.py', 'codeforces_58_B.py', 'atcoder_AGC007_C.py', 'codeforces_672_A.py', 'atcoder_ARC062_B.py', 'atcoder_AGC046_B.py', 'atcoder_ABC155_E.py'] 
Failed Test Details: ['Filename: codeforces_672_A.py Actual: 3 Generated: 952\n', 'Filename: atcoder_AGC007_C.py Actual: 649620280.95766068078000000000\n Generated: 649620280.9576600705197959310\n', 'Filename: atcoder_ARC062_B.py Actual: 0\n Generated: -1\n', 'Filename: codeforces_96_B.py Actual: 4444477777\n Generated: 7444447777\n', 'Filename: codeforces_58_B.py Actual: 733035 \n244345 \n48869 \n1  Generated: 733035 244345 48869 1 ', 'Filename: atcoder_ABC155_E.py Actual: 1\n Generated: 0\n', 'Filename: atcoder_AGC046_B.py Actual: 65\n Generated: 15\n'] 
=================================================================================================
Runtime Error Files: ['atcoder_ABC051_A.py', 'codeforces_369_B.py', 'atcoder_ABC169_C.py', 'codeforces_92_A.py', 'codeforces_99_A.py', 'atcoder_ABC114_C.py', 'atcoder_ABC168_C.py', 'codeforces_189_A.py'] 
Runtime Error Details: ['Filename: atcoder_ABC169_C.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/granite/avatar/java/python-sanitized/atcoder_ABC169_C.py", line 3, in <module>\n    a, b = map(int, sys.stdin.readline().split())\nValueError: invalid literal for int() with base 10: \'1.99\'\n', 'Filename: codeforces_99_A.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/granite/avatar/java/python-sanitized/codeforces_99_A.py", line 26, in <module>\n    sys.stdin, sys.stdout = open("input.txt", "r"), open("output.txt", "w")\nFileNotFoundError: [Errno 2] No such file or directory: \'input.txt\'\n', 'Filename: atcoder_ABC168_C.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/granite/avatar/java/python-sanitized/atcoder_ABC168_C.py", line 3, in <module>\n    a, b = map(int, input().split())\nValueError: too many values to unpack (expected 2)\n', 'Filename: codeforces_92_A.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/granite/avatar/java/python-sanitized/codeforces_92_A.py", line 20, in <module>\n    codeforces_92_A(sys.argv[1:])\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/granite/avatar/java/python-sanitized/codeforces_92_A.py", line 4, in codeforces_92_A\n    n, k = map(int, args)\nValueError: not enough values to unpack (expected 2, got 0)\n', 'Filename: atcoder_ABC051_A.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/granite/avatar/java/python-sanitized/atcoder_ABC051_A.py", line 3, in <module>\n    a, b, c = sys.stdin.readline().strip().split(",")\nValueError: too many values to unpack (expected 3)\n', 'Filename: codeforces_189_A.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/granite/avatar/java/python-sanitized/codeforces_189_A.py", line 17, in <module>\n    mcodeforces_189_Ain()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/granite/avatar/java/python-sanitized/codeforces_189_A.py", line 4, in mcodeforces_189_Ain\n    n = int(input())\nValueError: invalid literal for int() with base 10: \'29 27 18 2\'\n', 'Filename: atcoder_ABC114_C.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/granite/avatar/java/python-sanitized/atcoder_ABC114_C.py", line 26, in <module>\n    main()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/granite/avatar/java/python-sanitized/atcoder_ABC114_C.py", line 9, in main\n    ans += f(3 * 10**(len(s) - 1), len(s) - 2)\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/granite/avatar/java/python-sanitized/atcoder_ABC114_C.py", line 16, in f\n    return f(a + 3 * 10**m, m - 1) + f(a + 5 * 10**m, m - 1) + f(a + 7 * 10**m, m - 1)\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/granite/avatar/java/python-sanitized/atcoder_ABC114_C.py", line 16, in f\n    return f(a + 3 * 10**m, m - 1) + f(a + 5 * 10**m, m - 1) + f(a + 7 * 10**m, m - 1)\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/granite/avatar/java/python-sanitized/atcoder_ABC114_C.py", line 16, in f\n    return f(a + 3 * 10**m, m - 1) + f(a + 5 * 10**m, m - 1) + f(a + 7 * 10**m, m - 1)\n  [Previous line repeated 5 more times]\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/granite/avatar/java/python-sanitized/atcoder_ABC114_C.py", line 19, in f\n    if "3" in s and "5" in s and "7" in s and a <= n:\nNameError: name \'n\' is not defined\n', 'Filename: codeforces_369_B.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/granite/avatar/java/python-sanitized/codeforces_369_B.py", line 33, in <module>\n    main()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/granite/avatar/java/python-sanitized/codeforces_369_B.py", line 23, in main\n    n = int(scan.readline())\nValueError: invalid literal for int() with base 10: \'1 1 1 1 1 1\\n\'\n'] 
=================================================================================================
Compilation Error Files: ['codeforces_379_A.py', 'codeforces_55_A.py', 'codeforces_373_B.py', 'atcoder_ABC151_A.py', 'codeforces_86_A.py'] 
=================================================================================================
Infinite Loop Files: ['codeforces_669_A.py'] 
=================================================================================================
